util/paywall.go

```diff
@@ -15,6 +15,7 @@ import (
 	"github.com/decred/dcrd/dcrutil"
 	"github.com/decred/dcrd/hdkeychain"
 	"github.com/decred/dcrwallet/wallet/udb"
+	"github.com/decred/politeia/politeiawww/database"
```

Comment:
The `util` package shouldn't have any knowledge of `politeiawww`, so let's move this function into `politeiawww/backend.go` for now. I've added a new `politeiawww/paywall.go` file in my PR and this function really belongs there; I guess we can move it when mine is merged.

Reply:
fixed